charts/scorpio-broker-dist/values.yaml

```diff
@@ -0,0 +1,982 @@
+## namespace for deployment
+namespace: scorpio-broker
+
+################################### SpringArgs #######################################################################
+springArgs:
+  overrideSpringArgs: false
+  value: >
+
+################################### Eureka Server #######################################################################
+eureka:
+## Enable the eureka deployment & service
+  enabled: true 
+  name: eureka
```

Comment:
Whats the meaning of custom label? It would be good to use a mechanism like this for enabling custom labels:
https://github.com/FIWARE/helm-charts/blob/main/charts/orion/templates/deployment.yaml#L5-L6

Reply:
We have incorporated the review comment. Please review again